classes/classes/Scenes/Areas/GlacialRift/AmarokScene.as

```diff
@@ -0,0 +1,77 @@
+// By Foxwells
+// No wolf-fucking scenes because NO BESTIALITY. This is a WOLF and I am not having you fuck a LITERAL, NON-MORPH, ACTUAL WOLF.
+// Note: You get bad-ended if you lose 5 times in a row. Amarok eats you. I avoided vore fetish as best as I could (though I admit to making some miscellaneous jokes about it in commentary), did my best not to sexualise it. I don't like vore either. Shit's weird. Anyway if I crossed a line, let me know and I'll fix it.
+
+package classes.Scenes.Areas.GlacialRift 
+{
+	import classes.*;
+	import classes.GlobalFlags.kFLAGS;
+	import classes.GlobalFlags.kGAMECLASS;
+	
+	public class AmarokScene extends BaseContent
+	{
+		
+		public function AmarokScene() 
+		{
+		}
+
+		public function amarokChowTime():void {
+		//He's not actually going to eat you. Unless you fuck up 5 times.
+			clearOutput();
+			flags[kFLAGS.AMAROK_LOSSES]++;
+			if (doSFWloss()) {
+				return; //guess we should skip the vore in sfw mode, huh
```

Comment:
Does "return;" not force a stop to the code? My bad, then cwl Guess aS has more differences from C# than I thought.

Anyways, the SFW mode was first added because it was originally planned to add sex scenes of some sort, I grew too apathetic to write them and later learned bestiality isn't allowed anyway (thank God tbh). I've flat out removed it for now-- If I should re-add it and do something else, what do you suggest?